pom.xml

```diff
@@ -1457,6 +1525,14 @@
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
This will execute in every module, to ensure forward compatability, when new modules are introduced, they will not need to shade guava.
For modules that need to shade more than guava we should ask the question, should this be shaded in all modules that will use it? If so add it to the root pom, if not override shading for that specific module.